ubuntudesign/cloud-init.org
# Update favicon to have white bg

## Done
Updated favicon so it has greater contrast in various browser tabs.

## QA
Pull down code, run site, check the favicon in your browser tab has a white background.

Fixes: #37
